README.md

```diff
@@ -1713,6 +1713,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [Cronhooks](https://cronhooks.io/) - Schedule on time or recurring webhooks. Free plan allows 5 ad-hoc schedules.
   * [Hook Relay](https://www.hookrelay.dev/) - Add webhook support to your app without the hassles: done-for-you queueing, retries with backoff, and logging. The free plan has 100 deliveries per day, 14-day retention, and 3 hook endpoints.
   * [Format Express](https://www.format-express.dev) - Instant online formatter for JSON / XML / SQL.
+  * [Pika Code Screenshots](https://pika.style/templates/code-image) — Create beautiful, customizable screenshots from code snippets and VSCode using extension.
```

Comment:
Please mention "Free edition places a watermark on screenshots" or someting like that

Reply:
@ripienaar Actually this code screenshot template doesn't add any kind of watermark, even on free edition. I would like to keep it that way :)

Reply:
Maybe it doesnt, all I know is what the pricing page says so I go with that.

<img width="425" alt="Screenshot 2023-02-17 at 11 43 44" src="https://user-images.githubusercontent.com/82342/219622796-f21feb30-5d7c-471a-9d8d-77a4191391c2.png">

Reply:
@ripienaar Yeah, you're right, it's kind of confusing. I've tried to add more clarity about the watermarks in pricing page. It looks like this now:

<img width="468" alt="image" src="https://user-images.githubusercontent.com/14031295/219628465-1dd68573-c588-4ab5-bc83-24acaf23f405.png">

What do you think?